### PR TITLE
[Test] Fix windows chomedriver version to match linux workflow

### PR DESCRIPTION
**Pull request by joshuarrrr**
### Description

When we backported automation workflow for windows, we didn't ensure that the windows workflows use the same version as the linux workflows.
 
### Issues Resolved

Hopefully will fix all the failed windows workflows on the `2.x` branch 😓 
 
### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: .github/workflows/build_and_test_workflow.yml [hunk 1]
```diff
@@ -206,7 +206,7 @@ jobs:
 
       # image has the latest chrome v99
       - name: Setup chromedriver
-        run: yarn add --dev chromedriver@106.0.1
```
Comment (kavilla) [inline .github/workflows/build_and_test_workflow.yml:209]: the github public image uses chrome 99?

Comment (joshuarrrr) [reply .github/workflows/build_and_test_workflow.yml:209]: I don't know - but that's what line 157 says...

**Comment by AMoo-Miki** [comment]
> When we backported automation workflow for windows, we didn't ensure that the windows workflows use the same version as the linux workflows.

i am not sure I understand why they should use the same version. The linux workflow usees an image with an outdated chrome.
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/3ec4525eb70864a0e251611160aeadca0ea98ad7/.github/workflows/build_and_test_workflow.yml#L33

**Comment by joshuarrrr** [comment]
This is not necessary or useful or correct 😆

**Pull request closed without merging**